cmd/restic/cmd_prune.go

```diff
@@ -74,7 +74,7 @@ func init() {
 func addPruneOptions(c *cobra.Command, pruneOptions *PruneOptions) {
 	f := c.Flags()
 	f.StringVar(&pruneOptions.MaxUnused, "max-unused", "5%", "tolerate given `limit` of unused data (absolute value in bytes with suffixes k/K, m/M, g/G, t/T, a value in % or the word 'unlimited')")
-	f.StringVar(&pruneOptions.MaxRepackSize, "max-repack-size", "", "maximum `size` to repack (allowed suffixes: k/K, m/M, g/G, t/T)")
+	f.StringVar(&pruneOptions.MaxRepackSize, "max-repack-size", "", "stop after repacking this much data in total (allowed suffixes: k/K, m/M, g/G, t/T)")
```

Comment:
can we somehow keep the word `size` in backticks here? Cobra uses it to name the flag's parameter in the CLI help output. What about the following:
```
stop after repacking this data `size` in total (...)
```

Reply:
Or maybe 
```
stop after repacking data with this `size` in total
```